freqtrade/rpc/api_server/api_schemas.py

```diff
@@ -316,7 +316,7 @@ class ForceEnterPayload(BaseModel):
     entry_tag: Optional[str]
 
 
-class ForceExitPayload(BaseModel):
+class Force_exitPayload(BaseModel):
```

Comment:
We will not name classes like this.

Naming for Classes in python should be in `CamelCase` - which this was, therefore this is simply wrong.

Reply:
> `/force_exit` is a lot worse from a usability perspective (think phone) than `/forceexit` the moment you have to type it - therefore we'll surely not be accepting that part, as this will make usability worse, not better.

Added alias `/fe` as time is considered here.

Reply:
updated